cosmwasm/contracts/global-accountant/tests/chain_registration.rs

```diff
@@ -88,22 +88,30 @@ fn wrong_target() {
     body.payload.chain = Chain::Oasis;
 
     let (_, data) = sign_vaa_body(&wh, body);
-    contract
+    let err = contract
         .submit_vaas(vec![data])
         .expect_err("successfully executed chain registration VAA for different chain");
+    assert_eq!(
+        "this governance vaa is for another chain",
+        err.root_cause().to_string().to_lowercase()
+    );
 }
 
 #[test]
 fn non_governance_chain() {
     let (wh, mut contract) = proper_instantiate();
 
     let mut body = create_vaa_body();
+    body.payload.chain = Chain::Solana;
```

Comment:
why is this required? This tests the emitter chain check.

Reply:
I think I added this debugging, just removed.